scripts/gcs/flask_gcs.py

```diff
@@ -18,7 +18,7 @@ def __init__(self):
 		self.user_force_waypoint_update = self.manager.Value('i', False)
 
 		self.interop_client = self.manager.list()
-		self.interop_client.append(SUASSystem.InteropClientConverter())
+		#self.interop_client.append(SUASSystem.InteropClientConverter())
```

Comment:
Remember, we don't want to leave this line commented out when we push commits because it can lead to problems for others.

Reply:
Agreed.